tasks/calamari_setup.py

```diff
@@ -179,90 +218,117 @@ def calamari_install(config, cal_svr):
 
     The steps here are:
         -- Get the iceball, building it if necessary.
-        -- Copy the iceball to the calamari server, and untarring it.
-        -- Running ice-setup.py on the calamari server.
-        -- Running calamari-ctl initialize.
+        -- Copy the iceball to the calamari server, and untar/mount it.
+        -- Run ice-setup.py on the calamari server.
```

Comment:
perhaps drop the `.py` part here, since it's no longer a script?  The actual command now is `ice_setup`.